sh_scrapy/commands/shub_image_info.py

```diff
@@ -35,8 +36,21 @@ def add_options(self, parser):
     def run(self, args, opts):
         result = {
             'project_type': 'scrapy',
-            'spiders': sorted(self.crawler_process.spider_loader.list())
+            'spiders': sorted(self.crawler_process.spider_loader.list()),
+            'metadata': {},
         }
+        if sys.version_info >= (3, 8):
```

Comment:
I think here the condition could be "scrapy_spider_metadata import doesn't fail"